developer-docs-site/docs/tutorials/first-move-module.md

```diff
@@ -88,7 +88,7 @@ module HelloBlockchain::Message {
 }
 ```
 
-In the previous code, the two important sections are the struct `MessageHolder` and the function `set_message`. `set_message` is a `script` function allowing it to be called directly by transactions. Upon calling it, the function will determine if the current account has a `MessageHolder` resource and creates and stores the `message` it if it does not exist. If the resource exists, the `message` in the `MessageHolder` is overwritten.
+In this code, the two important sections are the struct `MessageHolder` and the function `set_message`. `set_message` is a `script` function allowing it to be called directly by transactions. Upon calling it, the function will determine if the current account has a `MessageHolder` resource and creates and stores the `message` it if it does not exist. If the resource exists, the `message` in the `MessageHolder` is overwritten.
```

Comment:
This usually refers to below?  Should we put it above the code?

Reply:
I've left it where it is and updated the text to say: "In the code above, ..."